automated-comments/rust/reverse-string/suggest_doing_bonus_test.md

```diff
@@ -0,0 +1,2 @@
+Consider expanding the solution to pass the feature-gated `test_grapheme_clusters` test case.
+Understanding how non-ASCII symbols can be handled and how to use external crates, has great benefits down the line.
```

Comment:
Reverse-string is the first exercise after hello-world, right? If so, it might be useful to add some links to pages detailing how to add external crates, and possibly even the crate itself or a hint on what the crate's name is.

Reply:
The hint on the crate name is already in the README: https://github.com/exercism/rust/blob/master/exercises/reverse-string/README.md#bonus

As for the learning link, how about this:
Understanding how non-ASCII symbols can be handled and how to [use external crates](https://doc.rust-lang.org/cargo/guide/dependencies.html), has great benefits down the line.

Reply:
I like the link! And thanks for the crate name explanation.